sections/collage.liquid

```diff
@@ -1,5 +1,6 @@
 {{ 'collage.css' | asset_url | stylesheet_tag }}
 {{ 'component-card.css' | asset_url | stylesheet_tag }}
+{{ 'component-badge.css' | asset_url | stylesheet_tag }}
```

Comment:
I personally wouldn't mind badge's styles just being moved into base, personally. Seems unnecessary to require a whole file for it.

Reply:
Yeah a request for 10 lines of CSS...